### PR TITLE
Add readme and coveralls

### PR DESCRIPTION
**Pull request by bess**
Work toward #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
-# README
+# pdc_discovery
 
-## Indexing a single handle
+A discovery portal for Princeton research data. Initially it will provide a better browsing experience for the research data containted in [DataSpace](https://dataspace.princeton.edu).
```
Comment (jrgriffiniii) [inline README.md:3]: `containted` was intended to be `contained` here

File: README.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+
+[![CircleCI](https://circleci.com/gh/pulibrary/pdc_discovery.svg?style=svg)](https://circleci.com/gh/pulibrary/pdc_discovery)
+[![Coverage Status](https://coveralls.io/repos/github/pulibrary/pdc_discovery/badge.svg?branch=main)](https://coveralls.io/github/pulibrary/pdc_discovery?branch=main)
```
Comment (jrgriffiniii) [inline README.md:6]: 👍

**Review by jrgriffiniii — CHANGES_REQUESTED** [review]
My only concern was one typo. error, but otherwise this is excellent

File: README.md (hunk 1 above)
Comment (bess) [reply README.md]: Fixed! Thank you!

**Review by jrgriffiniii — APPROVED** [review]

**Pull request merged**